### DISCUSSION TITLE
Running Kube-Bench

### DISCUSSION THREAD

**Discussion by adarshreddy24** [aquasecurity/kube-bench#1089] [Questions and Help] [answered]
How to run kube-bench with a binary downlaod?
How can we run a Kube-bench scan without Internet Access?

**Comment by mozillazg** [comment, marked as answer]
@adarshreddy24 

> How to run kube-bench with a binary downlaod? 

You can download the kube-bench from releases page: https://github.com/aquasecurity/kube-bench/releases

https://github.com/aquasecurity/kube-bench/blob/main/docs/installation.md#download-and-install-binaries

> How can we run a Kube-bench scan without Internet Access?

You can run kube-bench directly from the command line, and it can be run without internet.

https://github.com/aquasecurity/kube-bench/blob/main/docs/running.md#running-kube-bench
